tests/queries/0_stateless/02391_ReplacingCollapsingMergeTree.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE test (uid String, sign Int8, version UInt32) ENGINE = ReplacingCollapsingMergeTree(sign, version) Order by (uid);
+
+INSERT INTO test (*) VALUES ('d1', 1, 1), ('d2', 1, 1), ('d6', 1, 1), ('d4', 1, 1), ('d6', -1, 2), ('d3', 1, 1), ('d1', -1, 2), ('d5', 1, 1), ('d4', -1, 2), ('d1', 1, 3), ('d1', -1, 4), ('d4', 1, 3), ('d1', 1, 5);
```

Comment:
can we shuffle the input values so that we have:
```
('d1', 1, 3),('d1', -1, 2),('d1', 1, 1)
```?

Reply:
We can, I just wanted to check that if we insert values in "random" order, it doesn't affect the output.

Reply:
It can still be random, I was just curious about sorting (but it has to work otherwise next tests wouldn't work)